spec/models/voter_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe Vote, type: :model do
+  subject { create(:vote) }
+
+  it { is_expected.to have_attribute(:value) }
+
+  it { is_expected.to belong_to(:voteable) }
```

Comment:
nitpick: add space after this line since it's switching to a block-level (`it do`) syntax.